src/display_context.hpp

```diff
@@ -57,6 +57,8 @@ class display_context {
 	virtual const std::vector<std::string> & hidden_label_categories() const = 0;
 	std::vector<std::string> & hidden_label_categories_ref();
 	const team& get_team(int side) const;
+
+	template<typename T = void>
```

Comment:
If that's the problem, then it shouldn't be defined inline. Please move it to the source file.